components/rmrk/Profile/ProfileDetail.vue

```diff
@@ -313,6 +318,7 @@ export default class Profile extends mixins(
 
   public async mounted() {
     await this.fetchProfile()
+    this.fetchMyNftByIssuer()
```

Comment:
```suggestion
    await this.fetchMyNftByIssuer()
```

Reply:
I wonder why we need this `await`. Seems that there is no any function need to be called after this `this.fetchMyNftByIssuer()`

Reply:
good argumentation 👍